test/Combinatorics/IntegerCombinatorics/tableaux.jl

```diff
@@ -19,7 +19,6 @@
 	@test is_standard(Tableau([[1,2],[4]])) == false
 	@test is_standard(Tableau([[1,3,2],[4]])) == false
 
-
 	# is_semistandard
 	@test is_semistandard(Tableau([[1,2,4,7,8],[3,5,6,9],[10]])) == true
```

Comment:
please address the tabs here

Reply:
Right. I'll file that under the "little" things that need to be done for conformance with our style guide.